### ISSUE TITLE
[Item] Lord Valthalak's Amulet

### ISSUE THREAD

**Issue by Greywolf59** [WoWManiaUK/Redemption#5132]
**Links:**
Lord Valthalak's Amulet https://www.wow-mania.com/armory/?item=22048
Mea Culpa, Lord Valthalak https://www.wow-mania.com/armory/?quest=8995
Bodley https://www.wow-mania.com/armory/?npc=16033

**What is Happening:**
For Bodley NPC you do different quests to get different essences and left and right part of the amulet. When you get complete the amulet and grt to final quest you get to Mea Culpa quest thst requires yiu to slay Lord Valthalak and use the amulet on his body. Problem is that completed amulet somehow disappears when this quest is taken or when the previous one is handed in. I have no clue why and what is the precise problem is. However Bodley has chat option in which you tell him that either you lost one part of the amulet or the whole amulet and Bodley should give you the new one. But he does not. Nor giving the parts of the amulet works perfectly. He either give you only top one or left one but not both. This is near-end of very long and exhausting quest chain and it would be shame not being able to complete it

**What Should happen:**
I am not sure what should happen but either the amulet should not disappear or Bodley should give us the new one.

**Comment by Oganku** [comment]
I did that quest not too long ago and the amulet didn't disappear, but I'll double check.

**Comment by Greywolf59** [comment]
Well I still had this quest while doing this https://www.wow-mania.com/armory/?quest=8988
This "More components quests are available maybe out of the order in which they should. Cause when you talk to Bodley for the 1st time he has 4 types of "Components of Importance" and later on he opens one of the "More components..." to get more essence to summon another NPC for the next part od the amulet. I started this with 4th quest available as it was nearest aka the same quest as I linked here now except it was nit "More components" version but "Components" (1st quest) version. If that makes sense

**Comment by Rushor** [comment]
A reproduction step with the quest ids and i can fix it :)

**Comment by Greywolf59** [comment]
I don't remember it exactly (even though I done all that yesterday) but here goes nothing. Will do it in steps
1: Player finishes https://www.wow-mania.com/armory/?quest=8960 to find Bodley and get the quest Three Kings of Flames https://www.wow-mania.com/armory/?quest=8961.
2: Player hands in ?quest=8962 and Bodley opens 4 quests at once, all called Components of Importance
Those are:
https://www.wow-mania.com/armory/?quest=8962 for Mor Grayhoof branch
?quest=8963 for Isalien branch
?quest=8964 for  Jarien and Sothos
?quest=8965 for  Kormok

I chose 8965 for Kormok as it was nearest
3: Player does the chosen branch and gets left part of he amulet. He already has top one from quests before.
4: ater getting the left piece you get  https://www.wow-mania.com/armory/?quest=8970 and somewehere along the line of doing quest for left piece, at latest here, at the same tim with ?quest=8970 open 1st of possible quests called "More Components of Importance" and they are the same quests as the ones before, except instead of left part of the amulet you are doign the same for the right part.
5: Starting one of "More Components" leads to obtaining the right piece, fusing the amulet parts, empowering it, summoning and slaying Valthalak and using the amulet on his corpse. Now in the next comment I will try to remember what exactly I did.

**Comment by Greywolf59** [comment]
1: Picked up ?quest=8965
2A: Returned when done and picked up next part ?quest=8969
2B: I think ?quest=8986 was already avaialble and I picked it up
3: done 8969 and got left piece and handed the quest in
4: Picked up ?quest=8970, killed nagas for Bloodkelp. Not enough so I went to do ?quest=8986
5: I THINK I done ?quest=8986 and the next ?quest=8989 thus gettign right piece and fusing it into the whole amulet, not handing it in yet.
6: Finished 8970 and handed both 8970 and 8989
7: Handing in 8989 opened ?quest=8994, Also another "More Components" was available, this time ?quest=8987 and I took it
8: While waiting for the Flask required for 8994 I done 8987 and subsequent quest quest=8991 (although I am not sure if it actually was for the right part or the left part again, I think I had spare left part in the inventory  after doing it, which would mean it actually opened 8968 and I done that one).
9: I got the flast for 8994, handed it in, picked up  bugged quest Mea Culpa ?quest=8995, to slay lord Valthaalak and use the amulet on his body. At the same time after handing in the Left part (probably) from ?quest=8968 another "more Components" opened, this time https://www.wow-mania.com/armory/?quest=8988 which is basically the same quest line I did as the 1st one but before it was for left part, now it was for right part.
10: While having ?quest=8988 undone in quest log I went to slay Valthalak but I noticed I do not have the Amulet. Killed Valthalak but could not finished the quest cause the amulet was gone.
11: tried to talk to Bodley to give me new one, trashing the left peice I had and tried again, tried differnet combos really but amulet is not obrainable again for me.
12: Done that on Naxx as level 70 human warrior. Years ago I done that on Redemption as NElf druid lvl 80 with the same issue iirc.

**Comment by Oganku** [comment]
I will try when I get the chance, just haven't yet. Thanks for the steps!

**Comment by Rushor** [comment]
For me it works - maybe i understand it wrong?

if i have quest https://gyazo.com/1cb9c28f88eb7496379207d9f1bebded and talk to the Bodley, i get the item https://www.wow-mania.com/armory/?item=22048

https://gyazo.com/1cb9c28f88eb7496379207d9f1bebded

**Comment by Greywolf59** [comment]
Eh, okay. Let me check if I even have that chat option with him, cause there are diffrent chat options for me since I done the previous quests. I will post some SS after.

**Comment by Greywolf59** [comment]
So, yes. Following your example I got the amulet. The problem was I would never figure out to click "Bodley, tell me what's going on." Cause that option was always avaialble at the NPC and di not provide the amulet before, when clicking it.
Oh I can't put SS directly from laptop, well nvm then. I will just write it down. I have 3 chat options at Bodley.
1: I lost Lord Valthalak's Amulet!
2: I lost a piece of Lord Valthalak's Amulet!
3: Bodley, tell me what's going on.

I tried multiple times, multiple combos (quest, no quest, part of amulet, no part,...) clicking option 1 and 2. Option 1 did not provide me with amulet. Option 2 provided me either with top or left part, not with both, not with right (as if you'd have right part you'd actually have whole amulet already)
So the issues is that the wrong chat option provides the amulet. Thanks for all the help!
